pkg/Library/User/Profile.js

```diff
@@ -27,11 +27,16 @@ async onLogoutClick(inputs, state, {service}) {
 },
 
 onPersonaChange({eventlet: {value: persona}}) {
-  log(persona);
+  log(`persona = ${persona}`);
   return {persona};
 },
 
-render({persona, profile}, state) {
+onGroupChange({eventlet: {value: group}}) {
```

Comment:
`Profile` is a user login thing. Suggest making a new Particle ... 'LobbySettings' even that specific is ok.

It only needs to have the group label (technically) although any other iconography or whatever is cool.

Reply:
done